WordPress/WordPressUITests/Screens/ReaderScreen.swift

```diff
@@ -10,4 +10,8 @@ class ReaderScreen: BaseScreen {
 
         super.init(element: readerTable)
     }
+
+    static func isLoaded() -> Bool {
+        return XCUIApplication().tables["Reader"].exists
```

Comment:
Maybe we can have a `ElementStringIDs` struct/enum similar to the one declared on `MySiteScreen` to avoid repeating literal strings?

Reply:
Thanks for reviewing @etoledom ! I'll update the code to have it just like `MySiteScreen`